onboarder/installation/index.rst

```diff
@@ -53,8 +53,7 @@ Clear the Symfony cache and execute the following command to build the UI:
 
     rm -rf var/cache/*
     bin/console cache:warmup --env=prod
-    bin/console pim:installer:assets --clean --env=prod && yarn run webpack
-
+    make front
```

Comment:
The issue I see with `make front` is that it launches commands under Docker with `docker-compose` but customer could be directly on host and it will not work.
Also, from what I see in `front` target in the `Makefile` it build JS in dev mode (see `javascript-dev` target in the `Makefile`)

I'm not sure about `--symlink` option under production but what do you think of this instead:
`bin/console pim:installer:assets --symlink --clean --env=prod && yarn run less && yarn packages:build && yarn && yarn run webpack`

Reply:
Elodie told me she had access to the make commands. I will add a part without it, with the commands for v4 and v5

symlink not useful